src/main/navigation/navigation_fixedwing.c

```diff
@@ -238,8 +238,14 @@ static int8_t loiterDirection(void) {
     int8_t dir = 1; //NAV_LOITER_RIGHT
     if (pidProfile()->loiter_direction == NAV_LOITER_LEFT) dir = -1;
     if (pidProfile()->loiter_direction == NAV_LOITER_YAW) {
-        if (rcCommand[YAW] < -250) loiterDirYaw = 1; //RIGHT //yaw is contrariwise
-        if (rcCommand[YAW] > 250) loiterDirYaw = -1; //LEFT  //see annexCode in fc_core.c
```

Comment:
I agree with everything else but I kinda like these one line if statements for very simple thing. I think they improve the readability of the code.

Reply:
putting everything on one line won't make the algorithm any faster, I'm just following what's written here >> https://github.com/iNavFlight/inav/blob/master/docs/development/Development.md

Reply:
I disagree to leave it like this